nixos/modules/services/monitoring/loki.nix

```diff
@@ -22,6 +22,8 @@ in {
       '';
     };
 
+    package = lib.mkPackageOptionMD pkgs "grafana-loki" { };
```

Comment:
```suggestion
    package = mkPackageOptionMD pkgs "grafana-loki" { };
```

Reply:
This wouldn't work on its own since mkPackageOptionMD isn't in scope.

Reply:
oops, you are right.